client/dashboard/app/analytics/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { createContext, useContext } from 'react';
+
+export type AnalyticsClient = {
+	recordTracksEvent: ( eventName: string, properties?: Record< string, unknown > ) => void;
+	recordPageView: ( url: string, title: string ) => void;
```

Comment:
Why are `recordTracksEvent` and `recordPageView` separate functions?

Calypso records page views in both Tracks and Google Analytics. The v2 dashboard isn't using GA for now, but when the page view happens in the backport it will get recorded in both places.